static/src/javascripts/projects/common/modules/identity/consents.js

```diff
@@ -37,9 +37,6 @@ const UNSUBSCRIPTION_SUCCESS_MESSAGE =
     "You've been unsubscribed from all Guardian marketing newsletters and emails.";
 const ERR_MALFORMED_HTML = 'Something went wrong';
 
-const updateConsent = (consent: SettableConsent): Promise<void> =>
-    setConsent([consent]);
```

Comment:
I wasn't sure whether this added much over just inlining `setConsent([consent])` but happy to put back.